homeassistant/components/intellifire/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Intellifire Config",
```

Comment:
There's a linter that gets mad when you use a brand name in the title. You should just leave it out and it will use the title of the integration.